### PR TITLE
Updated add user modal for multitenancy admin

### PR DESCRIPTION
**Pull request by Bonymol-aot**
user_success

user_fail

### CODE REVIEW

File: forms-flow-admin/src/components/users/users.tsx [hunk 1]
```diff
@@ -372,22 +385,45 @@ const Users = React.memo((props: any) => {
...
-      toast.error(t("User not exist!"));
-    }
-  );
+    CreateUser(
+      payload,
+      (data) => {
+        openSuccessModal();
+      },
+      (err) => {
+        setValidationError(t('User not exist!'));
```
Comment (sumesh-aot) [inline forms-flow-admin/src/components/users/users.tsx:394]: Please change message to "User doesn't exist!"

File: forms-flow-admin/src/components/users/users.tsx [hunk 2]
```diff
@@ -372,22 +385,45 @@ const Users = React.memo((props: any) => {
...
+  
 
   return (
     <>
+      <Modal
+        show={showSuccessModal}
+        onHide={closeSuccessModal}
+        centered
+        className="overflow-hidden">
+        <Modal.Header closeButton>
```
Comment (sumesh-aot) [inline forms-flow-admin/src/components/users/users.tsx:407]: Please make sure the success modal is closed on clicking outside of the modal

Comment (Bonymol-aot) [reply forms-flow-admin/src/components/users/users.tsx:407]: Yes, its working so

File: forms-flow-admin/src/components/users/users.tsx (hunk 1 above)
Comment (Bonymol-aot) [reply forms-flow-admin/src/components/users/users.tsx:394]: Noted

**Review by shuhaib-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]

**Pull request merged**